src/index.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+//import * as serviceWorker from './serviceWorker';
+import Form from "./Form";
+
+
+function Worker() {
+    const [teams, setTeams] = useState("");
```

Comment:
Small thing here, but I'd recommend initialising the state here as an empty array since that's how you're storing data in it later on (as an array of objects with team data).